model/test_results.go

```diff
@@ -390,8 +395,8 @@ func FindTestResults(ctx context.Context, env cedar.Environment, opts TestResult
 		return nil, errors.New("cannot find with a nil environment")
 	}
 
-	if opts.TaskID == "" && opts.DisplayTaskName == "" {
-		return nil, errors.New("cannot find without a task_id or display_task_name")
+	if opts.TaskID == "" && opts.DisplayTaskID == "" {
```

Comment:
Should this also check for both being set, since [they're mutually exclusive](https://github.com/evergreen-ci/cedar/pull/316/files#diff-98447d1f255e456b4bdd74b07e213d03b003349d7113febe787beebeca269047R363-R366)?

Reply:
If both are set, it will use `TaskID`. Would you rather it error?

Reply:
I think it would make more sense to error since it basically just ignores the display task ID.